### ISSUE TITLE
[BUG] Error when paging via PagerDuty

### ISSUE THREAD

**Issue by sebastianrosch** [echoboomer/incident-bot#538]
**Describe the bug**
I integrated PagerDuty but receive the following error when paging someone (via Slack or when triggering an incident).

```
│ {"name": "root", "asctime": "2024-06-13 18:09:02,408", "levelname": "INFO", "filename": "incident.py", "message": "Request received from Slack to start a new incident: {'user': 'U04CXXXX', 'channel': 'modal',  │
│ 'incident_description': 'Test'}"}                                                                                                                                                                                  │
│ {"name": "root", "asctime": "2024-06-13 18:09:02,956", "levelname": "INFO", "filename": "incident.py", "message": "Creating incident channel: inc-202406131809-test"}                                              │
│ {"name": "root", "asctime": "2024-06-13 18:09:04,426", "levelname": "INFO", "filename": "incident.py", "message": "Sending message to digest channel for: inc-202406131809-test"}                                  │
│ {"name": "root", "asctime": "2024-06-13 18:09:05,707", "levelname": "INFO", "filename": "incident.py", "message": "Writing incident entry to database for inc-202406131809-test..."}                               │
│ {"name": "root", "asctime": "2024-06-13 18:09:06,242", "levelname": "INFO", "filename": "incident.py", "message": "Sending Statuspage prompt to inc-202406131809-test."}                                           │
│ {"name": "root", "asctime": "2024-06-13 18:09:06,242", "levelname": "INFO", "filename": "incident.py", "message": "Updating incident record in database with Statuspage message timestamp."}                       │
│ {"name": "root", "asctime": "2024-06-13 18:09:06,251", "levelname": "INFO", "filename": "incident.py", "message": "Paging awork..."}                                                                               │
│ {"name": "slack_bolt.App", "asctime": "2024-06-13 18:09:07,664", "levelname": "ERROR", "filename": "handler.py", "message": "Error: 'dict' object is not callable", "exc_info": "Traceback (most recent call last) │
│ :\n  File \"/usr/local/lib/python3.11/site-packages/slack_bolt/listener/thread_runner.py\", line 120, in run_ack_function_asynchronously\n    listener.run_ack_function(request=request, response=response)\n  Fil │
│ e \"/usr/local/lib/python3.11/site-packages/slack_bolt/listener/custom_listener.py\", line 50, in run_ack_function\n    return self.ack_function(\n           ^^^^^^^^^^^^^^^^^^\n  File \"/incident-bot/bot/slack │
│ /modals.py\", line 377, in handle_submission\n    resp = incident.create_incident(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/incident-bot/bot/incident/incident.py\", line 421, in create_incident\n    asy │
│ ncio.run(\n  File \"/usr/local/lib/python3.11/asyncio/runners.py\", line 190, in run\n    return runner.run(main)\n           ^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/asyncio/runners.py\", line 118, │
│  in run\n    return self._loop.run_until_complete(task)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/asyncio/base_events.py\", line 654, in run_until_complete\n    return  │
│ future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/incident-bot/bot/incident/incident.py\", line 600, in handle_incident_optional_features\n    pd_api.page(\n  File \"/incident-bot/bot/pagerduty/api.py\", l │
│ ine 144, in page\n    service_id = find_service_for_escalation_policy(ep_name=ep_name)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/incident-bot/bot/pagerduty/api.py\", line  │
│ 56, in find_service_for_escalation_policy\n    if ep(\"name\") == ep_name:\n       ^^^^^^^^^^\nTypeError: 'dict' object is not callable"}
```

**Version**
v1.10.7

**To Reproduce**
1. Configure PagerDuty integration
2. Notice incidentbot is able to fetch schedules: `│ {"name": "root", "asctime": "2024-06-13 18:04:33,936", "levelname": "INFO", "filename": "api.py", "message": "PagerDuty returned 1 schedules"}`
3. In the UI, select a schedule to page
4. Create an incident via Slack
5. Notice the error in the logs

**Expected behavior**
The incident creation should not result in an error and the page should be sent via PagerDuty.

**Screenshots**
<img width="1623" alt="Screenshot 2024-06-13 at 20 06 37" src="https://github.com/echoboomer/incident-bot/assets/2076369/5fac0899-8fc3-4cfc-8c2b-fe6f6378195a">
<img width="2128" alt="Screenshot 2024-06-13 at 20 14 06" src="https://github.com/echoboomer/incident-bot/assets/2076369/ffb9d4fa-b39a-4bab-96ac-039e850c7224">
<img width="1599" alt="Screenshot 2024-06-13 at 20 14 16" src="https://github.com/echoboomer/incident-bot/assets/2076369/52167482-1da3-4043-90df-a805d97e95d3">
<img width="582" alt="Screenshot 2024-06-13 at 20 18 17" src="https://github.com/echoboomer/incident-bot/assets/2076369/1724f7d2-4427-40f6-8403-fcfe6f339baf">

**Environment (please complete the following information):**
 - Platform: Kubernetes

**Additional context**

**Comment by echoboomer** [comment]
@sebastianrosch I just released `v1.10.8` - would you mind bumping and seeing if this issue is resolved on your end?

**Comment by sebastianrosch** [comment]
Thank you so much @echoboomer, it works now!
